src/work.c

```diff
@@ -96,7 +96,7 @@ static void luv_work_cb(uv_work_t* req) {
 
   if (lua_isfunction(L, -1)) {
     int i = luv_thread_arg_push(L, &work->arg, 0);
-    i = luv_cfpcall(L, i, LUA_MULTRET, LUVF_CALLBACK_NOEXIT);
+    i = luv_cfpcall(L, i, LUA_MULTRET, 0);
```

Comment:
I think this can still pass `LUVF_CALLBACK_NOEXIT`, since that matches how it worked in 1.9.1:

https://github.com/luvit/luv/blob/976c886cb4ade983dadedc721d9f0071924d5aee/src/work.c#L99-L105

It might lead to the same sort of problems described in https://github.com/luvit/luv/issues/433#issuecomment-548973999 but I'm not familiar enough with the threading side of Luv/Libuv to know.

Reply:
On second thought, it's probably better to exit here too. Printing an error and continuing without the user being able to react to the error is probably always a bad idea.